pom.xml

```diff
@@ -64,7 +64,13 @@
             <version>1.2.83</version>
         </dependency>
 
-    </dependencies>
+        <!-- text -->
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-text</artifactId>
```

Comment:
Can you implement `StringEscapeUtils.unescapeJava()` inside this project to avoid extra dependency?